### ISSUE TITLE
Cannot start a new line by pressing enter in android

### ISSUE THREAD

**Issue by Cateners** [godotengine/godot#72105]
### Godot version

v4.0.beta15.official [4fa6edc88]

### System information

Android 12

### Issue description

I simply created a TextEdit node, ran the scene and everything worked fine on my computer (Windows 11) :
![20230126_183710 00_00_00-00_00_30](https://user-images.githubusercontent.com/58554009/214815786-1e43510f-6442-4676-9629-6df4b1c5b71a.gif)

Then I made an android debug build and installed it and ran
However, I couldn't start a new line by pressing enter:
![VID_20230126_173711 00_00_02-00_00_11](https://user-images.githubusercontent.com/58554009/214815839-330aa2dd-9272-4c59-9501-e7a957533b6e.gif)

It won't work even using a physical keyboard:
![VID_20230126_173711 00_00_14-00_00_23](https://user-images.githubusercontent.com/58554009/214815886-7c0f992e-44b3-40ce-81e8-f08c32456678.gif)

I've tried it on both my tablet (Android 10) and my phone (Android 12) . And they both have the above mentioned problems.

### Steps to reproduce

Mentioned above; Just need a TextEdit node to test it out.

### Minimal reproduction project

N/A

**Comment by m4gr3d** [comment]
I'm running into the same issue in the Android Editor for Godot 4. Could it be related to https://github.com/godotengine/godot/pull/70052?

cc @bruvzg

**Comment by m4gr3d** [comment]
Yeah, the regression starts after https://github.com/godotengine/godot/pull/70052
